extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -388,9 +389,13 @@ function createDicomWebApi(dicomWebConfig, servicesManager) {
                 // isn't well specified in the standard, but is needed in
                 // any implementation that stores static copies of the metadata
                 StudyInstanceUID: naturalized.StudyInstanceUID,
+                mediaTypes: mediaType
```

Comment:
This change allow requesting first in the provided media type, and then defaulting to application/octet-stream if not found.